### DISCUSSION TITLE
OSRS Bot COLOR | Python Botting Toolkit w/ Custom UI for Old School RuneScape & RSPS

### DISCUSSION THREAD

**Discussion by kelltom** [TomSchimansky/CustomTkinter#456] [Showcase]
🔹This is a portfolio project, so likes/comments/stars on the YouTube video and repository are hugely appreciated!🔹

OSRS Bot COLOR (OSBC) is a lightweight desktop app for controlling and monitoring color-based automation scripts (bots) for Old School RuneScape and private servers. This project also contains a library of tools for streamlining the development of new bots and enhancing their capabilties. OSBC allows developers to leverage computer vision (optical character recognition, image recognition, color isolation, etc.) through a custom library to enhance bot capabilities.

Massive 'thank you' goes out to everyone responsible for CustomTkinter!

📹 [Watch the showcase on YouTube](https://www.youtube.com/watch?v=DAbyiW5mY3M)

⭐ [Leave a Star on GitHub](https://github.com/kelltom/OSRS-Bot-COLOR)

![title](https://user-images.githubusercontent.com/44652363/190813767-1daba8a3-9a90-48cf-ac38-32781434d995.png)

![menu](https://user-images.githubusercontent.com/44652363/190813789-2257501d-fee7-4759-b6cc-dbd4d394ae3d.png)

**Comment by TomSchimansky** [comment]
Looks great, makes me happy to see the library in use for bigger projects like this 👍.

**Comment by sumeshir26** [comment]
Ooh looks cool!

**Reply by kelltom** [reply]
Thank you!
